pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -378,6 +378,17 @@
      */
     ConsumerBuilder<T> acknowledgmentGroupTime(long delay, TimeUnit unit);
 
+    /**
+     * Group the consumer acknowledgments for size.
+     *
+     * <p>By default, the consumer uses 1000 messages to send out the acknowledgments to the broker.
+     *
+     * @param messageNum
+     *
+     * @return the consumer builder instance
+     */
+    ConsumerBuilder<T> acknowledgmentGroupSize(int messageNum);
```

Comment:
How about `maxAcknowledgmentGroupSize(int maxAckGroupSize)`

First we should clarify that it is a max size in the method  name,  and then it's better  keep the params consistent with the field to be set. WDYT:)

Reply:
OK

Reply:
Fixed.
PTAL,thanks! @AnonHxy